python/paddle/autograd/saved_tensors_hooks.py

```diff
@@ -103,7 +108,11 @@ class saved_tensors_hooks:
             >>> y.sum().backward()
     """
 
-    def __init__(self, pack_hook, unpack_hook):
+    def __init__(
+        self,
+        pack_hook: Callable[[Tensor], Tensor | None],
+        unpack_hook: Callable[[Tensor], Tensor | None],
```

Comment:
```suggestion
        pack_hook: Callable[[Tensor], Any | None],
        unpack_hook: Callable[[Any], Tensor | None],
```

文档中的意思应该是这样？

另外，下面的两个方法也标一下吧 ～

Reply:
收到